Makefile

```diff
@@ -25,7 +25,7 @@ OS_DIST ?= $(shell rpm --eval='%dist')
 
 PYTHON_VER ?= $(shell $(PYTHON) -c 'import sys; print("python%s.%s" % sys.version_info[:2])')
 
-PYTHON_SITELIB ?= $(PREFIX)/lib64/$(PYTHON_VER)/site-packages
+PYTHON_SITELIB ?= $(PREFIX)/lib/$(PYTHON_VER)/site-packages
```

Comment:
I remember reviewing [a PR](https://github.com/candlepin/subscription-manager/pull/1823/files) that did the exact opposite of this (changed lib to lib64) because Anaconda wouldn't display the subman addon. Maybe there is a way to better decide where to install, but I have no idea what is the right thing to do here (i.e. where lib or lib64 is/should be used). Any thought on this @jirihnidek ?

Reply:
Maybe we can use `/usr/lib/$(PYTHON_VER)/site-packages` only for dnf plugin?

Reply:
@jirihnidek 's solution makes sense to me, its kind of what I was working on locally
in the current work actually, what ever we set here is ignored, since spec file overrides the variable.
I will add another variable for plugins

Reply:
:+1: